apps/wallet/browser/kernel/utils/helper.ts

```diff
@@ -13,11 +17,37 @@ export const addExtensionsById = async (id: string) => {
 		return;
 	}
 
-	await modules.storage.put<ExtensionDocument>(extensionDoc);
+	await storage.put<ExtensionDocument>(extensionDoc);
 };
 
 export const checkInstalledExtensionById = async (id: string) => {
-	const extension = await modules.storage.safeGet<ExtensionDocument>(id);
+	const extension = await storage.safeGet<ExtensionDocument>(id);
 
 	return extension ? true : false;
 };
+
+export const settings = {
```

Comment:
this exported `settings` var name is not clear to me

Reply:
It was a mock setting for a long time, I just keeping it